### ISSUE TITLE
Specific message links don't open the full conversation

### ISSUE THREAD

**Issue by M86xKC** [Tunous/Dawn#344]
**Describe the bug**
---
When opening links of specific (nested) comments without the appending `/?context=3` to the URL it only loads the specfic comment but not the (parent/previous) message(s), I don't know if this is intentional or not?

When using an RSS feed it does not append the `/?context=3` to nested comments, so Dawn only loads that last specific comment without the context (parent/previous) messages

**To Reproduce**
---
1. Go to `your RSS reader of choice` 
2. Subscribe to `/u/exmaple/.rss`
3. open a nested comment from the RSS reader app. [example](https://reddit.com/r/Android/comments/k0ynjj/ama_were_lineageos_developers_of_the_most_popular/gdl64wz/)
4. If the comment is not a **top-level/parent** comment it only show that last specific comment! with no context/previous comments.

**URLs of affected threads**
---
https://reddit.com/r/Android/comments/k0ynjj/ama_were_lineageos_developers_of_the_most_popular/gdl64wz/

**Expected behavior**
---
IMO it is necessary in circumstances like this for the message thread to be loaded no matter if  `/?context=3` is present or not.

**Smartphone (please complete the following information):**
---
 - Device: no applicable
 - OS: no applicable
 - Version: 0.11.0

**Comment by AbsurdlySuspicious** [comment]
Indeed, 3-post context would be a sane default for links without it.

(Note to implementor) I believe links where context is already specified should be left as is no matter what its value is
